hibernate-core/src/main/java/org/hibernate/jpa/internal/enhance/EnhancingClassTransformerImpl.java

```diff
@@ -32,12 +32,13 @@ public class EnhancingClassTransformerImpl implements ClassTransformer {
 	private volatile WeakReference<Entry> entryReference;
 
 	//This list is matching the constants used by CoreTypePool's default constructor
-	private static final String[] NO_TRANSFORM_PREFIXES = { "jakarta/", "java/", "org/hibernate/annotations/" };
+	private static final String[] NO_TRANSFORM_PREFIXES = { "jakarta/", "java/", "org/hibernate/annotations/", "org/hibernate/bytecode/enhance/spi/", "org/hibernate/engine/spi/" };
```

Comment:
Shouldn't this be made public and re-used directly in `CorePrefixFilter`'s default constructor?

Reply:
I agree _something_ can be improved here but I wonder how much it's a distraction: their purpose is slightly different and the format of the strings is also different.